AgentDiscoveries-Backend/src/main/resources/db/migration/V9__add_long_lat.sql

```diff
@@ -0,0 +1,6 @@
+ALTER table locations
+ADD COLUMN longitude decimal(9,7) AFTER site_name,
+ADD COLUMN latitude decimal(9,7) AFTER longitude;
+UPDATE locations
+SET longitude = 90.293428, latitude = 08.2398472
+WHERE location_id = 1;
```

Comment:
It's weird to be updating data in a migration, but it's also weird that we chose to add data in a migration at the start, so I'm happy with this but should flag it for learning purposes!